back-office/shop-parameters/order-settings/order-setting-statuses-page.md

```diff
@@ -1,48 +1,167 @@
 
+When clicking on it, three options are displayed:
 
+1) Refresh list
 
-*   **Id,**
-*   **name with their color label,**
-*   **icon (to be removed after migration since it is not used anymore),**
-*   **send email to the customer,**
-*   **in transit,**
-*   **invoice,**
-*   **email template,**
-*   **edit button.**
+After clicking ont it, the whole page is refreshed.
 
-![Order statuses list](/img/list%20order%20statuses.png)
+2) Show SQL query
+
+After clicking on it, the SQL query is displayed in a modal. The SQL query takes into account filters and sorts that have been done.
+
+There are two buttons:
+
+- Close: After clicking on it, the modal is closed
+- Export to SQL Manager: After clicking on it, the SQL query is opened in Advanced parameters > Database > SQL Manager
+
+3) Export to SQL Manager
+
+After clicking on it, the SQL query is opened in Advanced parameters > Database > SQL Manager
+
+### Bulk actions drop-down
+
+This dropdown is enabled only if at least one order status is selected, otherwise it's disabled.
+
+There is 1 option:
+
+1) Delete selection
+
+After clicking on it, a modal is opened asking to confirm or cancel the action: "Are you sure you want to delete the selected item(s)?"
+
+The action can be canceled by clicking on the cross or on the cancel button.
+
+After clicking on "Delete" button, the selected order statuses are deleted, a successful message is displayed "The selection has been successfully deleted."
+
+### Listing
+
+The **order statuses** list displays all the order statuses.
+
+#### Multistore behavior
+
+The whole listing page can be displayed only in all shops context, the others contexts are not displayed in the header.
+
+If the user was on another page in a specific shop context or in a group context, the all shops context is automatically selected when he/she arrives on the page.
+
+An information message is displayed at the top of the page: "Note that this page is available in all shops context only, this is why your context has just switched." (_See issue [#19495](https://github.com/PrestaShop/PrestaShop/issues/19495)_)
+
+:point_right: https://github.com/PrestaShop/prestashop-specs/blob/master/broader-topics/multistorespecialsspecs.md
+
+#### Select all / unselect all checkbox
+
+After checking the box, all the order statuses with a checkbox are selected or unselected.
+
+#### Columns
+
+1) Checkbox: The checkbox isn't displayed for the default order status and the status added by modules.
+
+There are 8 columns in the grid:
+
+1) Id
+
+It's possible to sort by id asc or desc and search by id.
+
+2) Name 
+
+It's possible to sort by name asc or desc and search by name.
+
+The order status names are displayed with their color in background.
+
+3) Icon 
+
+This should be removed during the migration since it is not used anymore.
+
+4) Send email to the customer
+
+The merchant can choose to send an email or not to the customer when the order status is applied to the order.
```

Comment:
If no mail template is selected for the status, should the merchant still be able to change the "Send email" setting to Yes ? (Currently, doing this will cause an error on an order if the status is changed to the status with "Send mail" to "Yes" and no template)

Reply:
No it should not, we need to specify the behavior + maybe a message in this case
Do you know if there is an issue for that ? I can link it to the specs but I didn't find any issue

Reply:
Done here: https://github.com/PrestaShop/prestashop-specs/pull/207/commits/76c9687e0c0db09ff3d9d8f888274d99dfd482a9